frontend/app/work_packages/controllers/work-package-details-controller.js

```diff
@@ -172,13 +174,78 @@ module.exports = function($scope,
     return !!($scope.workPackage && $scope.workPackage.embedded.watchers !== undefined);
   };
 
-  function displayedActivities(workPackage) {
+  $scope.isInitialActivity = function(activity, activityNo) {
+    var type = activity.props._type,
+      activities = $scope.activities;
+
+
+    // Type must be Activity
+    if (type !== 'Activity') {
```

Comment:
`&& type !== 'Activity::Comment'` -.-

I always thought that this differentiation sucks... but it is there and apparently it is used in the angular client as well...

Noticed that this is neccessary, because of a first journal of mine which is tagged as `Activity::Comment`.